pkgs/stdenv/adapters.nix

```diff
@@ -90,7 +94,6 @@ rec {
                     name = name + "-" + cross.config;
                     nativeBuildInputs = nativeBuildInputsDrvs
                       ++ nativeInputsFromBuildInputs
-                      ++ [ gccCross binutilsCross ]
```

Comment:
Ah, well for iOS we do. And the normal stdenv doesn't just pick them up from build packages yet because all the stdenvs predate that.